### DISCUSSION TITLE
Upgrading from 2.9.75 to 2.10.12 - As easy as it looks?

### DISCUSSION THREAD

**Discussion by jengstrom440** [semaphoreui/semaphore#2162] [Q&A] [answered]
So to upgrade Semaphore to the latest all one needs to do is install the new package like the previous one was? If so, that is great news. Any gotchas one should be aware of?

**Comment by fiftin** [comment, marked as answer]
Hi @jengstrom440 
Yes, just install the new package, but don’t forget to back up the database. ;)

**Reply by jengstrom440** [reply]
That's great news, thanks for making it easy! I appreciate the efforts put into this product, it's been great not having to deal with containers.

**Comment by jengstrom440** [comment]
As a follow-up, I updated to the latest release today without issue after backing up the DB. After rebooting the server, it seems to work fine. I am looking forward to trying the new features, especially PowerShell!
